src/utils.js

```diff
@@ -0,0 +1,3 @@
+export function batchRunLabel(br) {
+    return !!br ? `${br.runDate.substr(0,br.runDate.indexOf('T'))} ${br.location.code} ${br.location.name}: ${br.runYear}-${br.runMonth}` : "";
```

Comment:
`substr` is deprecated: https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Global_Objects/String/substr
Please use `substring` instead. Also you could rename the function to `getBatchRunLabel` - the more accurate the names the easier it will be for other developers to read the code.

Reply:
sure! thx for giving me link about this. I will read more about this. Next time I will check if external function that I use is still supported etc. 

Regarding changing name - ok, I will change. You are right about this.